test/controllers/jobs_controller_test.rb

```diff
@@ -1,7 +1,9 @@
 require 'test_helper'
 
 class JobsControllerTest < ActionController::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  include Devise::TestHelpers
```

Comment:
@godfreym it is better you put this line in the test_helper, otherwise you will have to include it in every controller test